scripts/footprint/size_report

```diff
@@ -635,8 +636,11 @@ def main():
         if args.json:
             exporter = JsonExporter(indent=2, sort_keys=True)
             data = exporter.export(root)
+            target = dict()
+            target["symbols"] = json.loads(data)
```

Comment:
Why not change the exporter in the line above to be a dict exporter ?

That way you don't need to export to json and load into a dict, just to reformat back to json  at line 643 when doing `json.dump`.

Basically, what i'm proposing is:
```
            exporter = DictExporter()
            data = dict()
            data["symbols"] = exporter.export(root)
            data["total_size"] = symsize
            with open(args.json, "w") as fp:
                json.dump(data, fp, indent=4)
```

In case you do this change, remember to update line 33 to:
```
from anytree.exporter import DictExporter
```

Reply:
OK, thank you for your suggestion.